### ISSUE TITLE
Sprite.Events 'change' Stroke event triggered only by first stroke. 

### ISSUE THREAD

**Issue by DMunkei** [aseprite/aseprite#3787]
Creating a script to trigger on each sprite change. Triggers only on initial stroke and doesn't trigger for strokes after.

### Aseprite and System version
V1.2.40-dev
5.15.104-2-MANJARO 
Sample code to reproduce
```lua
specs = ImageSpec{
    width=512,
    height=512,
    }
local sprite = Sprite(specs)
local function func()
    print('change in sprite')
end
sprite.events:on('change', func)
```
Expected: is every stroke triggers the console. 
Actual: Triggers once and doesn't trigger console after first stroke.

It only triggers, if I change to a different tool and try a different stroke. 

Example code is following the code from the docs found here:
[Sprite events](https://www.aseprite.org/api/sprite#spriteevents)

**Comment by dacap** [comment]
Hi @DMunkei, I've tested on v1.2.40 and v1.3-rc2 and I think the events are still received, but if you close the console, following messages aren't printed until a new console is opened.

There is a bug where the console doesn't pop up until you change to other sprite and go back to the original sprite.

I'll take a look to this. Anyway if you keep drawing without closing the console (the console is not a modal dialog) you will see that you receive a change event for each stroke.

**Comment by dacap** [comment]
Possible fix in be11e3b32429720d94d38357788020571897816a, closing this issue, but feel free if it can be reproduced in v1.3-rc3 when it's released.
